gtsam/3rdparty/cephes/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(
 
 set(CEPHES_HEADER_FILES
     cephes.h
+    dllexport.h
```

Comment:
You can undo this and other related changes (aka `git revert`) and then rebase the latest develop after #1704 is merged.

Reply:
I will rebase your PR and then undo / resolve the conflicts.